bundle/Form/DataMapper/InformationCollectionUpdateMapper.php

```diff
@@ -0,0 +1,78 @@
+<?php
+
+namespace Netgen\Bundle\InformationCollectionBundle\Form\DataMapper;
+
+use Ibexa\Contracts\Core\Repository\Values\ContentType\ContentType;
```

Comment:
This is not used, please run CS fixer on the changed files.

Reply:
issue fixed in a5f8e50 commit